images.CI/download-customrepo.ps1

```diff
@@ -0,0 +1,14 @@
+param(
+    [String] [Parameter (Mandatory=$true)] $CustomRepoUrl,
+    [String] [Parameter (Mandatory=$true)] $CustomRepoBranch
+)
+
+Write-Host "Clean up default repository"
+Remove-Item -path './*' -Recurse -Force
+
+Write-Host "Download ${CustomRepoBranch} branch from ${CustomRepoUrl}"
+$env:GIT_REDIRECT_STDERR = '2>&1'
+git clone $CustomRepoUrl . -b $CustomRepoBranch --single-branch --depth 1
+
+Write-Host "Latest commit:"
```

Comment:
Can we reuse Get-LatestCommit here?

Reply:
We can, but Get-LatestCommit function was created only in helper scripts that are completely separated from our Images.CI definitions, and it will doesn't make sense to have common helpers for both images.CI and helpers.